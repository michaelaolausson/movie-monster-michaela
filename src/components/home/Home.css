
.start-container, .start-container-background {
    position:absolute;
    height: 80vh;
    top: 20vh;
}
.start-container {
    z-index: 2;
}
.start-container .row {
    height: 100%;
    align-items: center;
}
.start-container-background {
    content: ' ';
    display: block;
    left: 0;
    right: 0;
    z-index: 1;
    opacity: 0.6;
    background-image: url(https://image.tmdb.org/t/p/original/3p5YkOrajk3DVYrGwX64p8ervti.jpg);
    background-position: center;
    -ms-background-size: cover;
    -o-background-size: cover;
    -moz-background-size: cover;
    -webkit-background-size: cover;
    background-size: cover;
}
h2,p {
    background: #333;
    padding: 1rem;
}

@media screen and (max-width: 992px) {
    .start-container,.start-container-background { 
        top: 0vh;
        height: 90vh;
    }

}
