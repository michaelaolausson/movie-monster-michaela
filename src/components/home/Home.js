import React from "react";
import { Container, Row, Col } from "react-bootstrap";
import "./Home.css";

const Home = () => {
  return (
    <div>
      <Container className="start-container">
        <Row>
          <Col>
            {localStorage.username && <h2>Welcome back to Movie Monster</h2>}
            {!localStorage.username && <h2>Welcome to Movie Monster</h2>}
            <p>The only movie finding service you'll ever need</p>
          </Col>
        </Row>
      </Container>
      <Container className="start-container-background"></Container>
    </div>
  );
};

export default Home;
