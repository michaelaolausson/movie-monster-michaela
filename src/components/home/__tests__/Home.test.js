import React from "react";
import "@testing-library/jest-dom/extend-expect";
import { shallow } from "enzyme";
import Home from "../Home";

describe("<Home />", () => {
  const wrapper = shallow(<Home/>);
  it("should match the snapshot", () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});