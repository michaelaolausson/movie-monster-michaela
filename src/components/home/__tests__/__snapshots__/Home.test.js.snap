// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should match the snapshot 1`] = `"<div><div class=\\"start-container container\\"><div class=\\"row\\"><div class=\\"col\\"><h2>Welcome to Movie Monster</h2><p>The only movie finding service you&#x27;ll ever need</p></div></div></div><div class=\\"start-container-background container\\"></div></div>"`;
